src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -765,6 +765,14 @@
           "for at most jint_max / StressLongCountedLoop")                   \
           range(0, max_juint)                                               \
                                                                             \
+  product(bool, DuplicateBackedge, true,                                    \
```

Comment:
Why flag is `product`?  Can it be `experimental` or `diagnostic`? I assume eventually we should remove this flag.

Reply:
Right. I made it diagnostic in the new commit.